homeassistant/components/alert/services.yaml

```diff
@@ -18,3 +18,33 @@ turn_on:
   target:
     entity:
       domain: alert
+
+notification_control:
```

Comment:
So we already have services for turning it on and off, for those cases, we don't need to add fields in this service call. This leaves the `snooze` capability.

I suggest renaming the service to `alert.snooze` instead, accepting the optional time it can be snoozed (in time format, not hours, so it becomes more flexible).

Reply:
Thanks for giving it a look. `alert.turn_on` & `alert.turn_off` affect only the current firing of the alert (i.e., until the condition for the alert becomes false).

The `notification_control` call persists through future alert firings. It's useful if the condition for an alert alternates between true and false - e.g., a water sensor that's slightly wet or a temperature sensor that hovers around a threshold.

I wish I had a better name for it. In an ideal world, to be more precise, we might rename the services: 
- `alert.turn_off` -> `alert.acknowledge`
- `alert.turn_on` -> `alert.unacknowledge`

and then add alert.notifications_on, alert.notifications_off and alert.notifications_snooze to capture the new behavior.

I've changed snooze to take a datetime as you suggest, and updated the comments to hopefully be a bit clearer.